### ISSUE TITLE
Техническая ошибка при запуске токенизации

### ISSUE THREAD

**Issue by kon3gor** [yoomoney/yookassa-android-sdk#73]
Добрый день, пытаюсь подключать сдк, но в BottomSheet после запуска постоянно отображается сообщение 
__Seems we've had a technical error__, а в логах вообще ничего нет, несмотяр на то, что в тестовых параметрах стоит **showLogs = true**
Ниже скрин того, как я создаю PaymentParameters
![image](https://user-images.githubusercontent.com/37112953/113752913-2d4c9380-9716-11eb-8081-c2bdc039c097.png)
И вот какой BottomSheet вылезает
![image](https://user-images.githubusercontent.com/37112953/113752824-15750f80-9716-11eb-8653-51518ed222de.png)

**Comment by iskandarshabaev** [comment]
Добрый день. Напишите пожалуйста нам на почту b2b_support@yoomoney.ru о своей проблеме, поможем разобраться. Просьба в письме указать на чём запускается приложение устройство/эмулятор, версию android и shopId и clientId.

**Comment by iskandarshabaev** [comment]
Кажется проблема в нулевой сумме ```new Amount(BigDecimal.ZERO, Currency.getInstance("RUB"))```
Попробуйте передать  например 10

**Comment by iskandarshabaev** [comment]
Над логированием уже работаем, выкатим в ближайшем релизе.

**Comment by kon3gor** [comment]
> Кажется проблема в нулевой сумме `new Amount(BigDecimal.ZERO, Currency.getInstance("RUB"))`
> Попробуйте передать например 10

Нет, я ставил, и ноль, и десять, и вообще рандомную стоимость, результат не изменился
